docs/SPEC.md

```diff
@@ -41,6 +41,11 @@ the API or data source. The format of the configuration will vary by Tap,
 but it must be JSON-encoded and the root of the configuration must be an
 object.
 
+For taps where the credentials need to be refreshed before or during a
+run, the tap can write changes to the config file, and the changes will be
+saved and picked up on subsequent runs. The singer-python method
+`update_config_file()` can be used to make the necessary changes.
```

Comment:
The last sentence here can be removed. The spec shouldn't contain specific function calls.